tests/framework/utils/helm_helper.go

```diff
@@ -62,7 +62,12 @@ func (h *HelmHelper) GetLocalRookHelmChartVersion(chartName string) (string, err
 	}
 	cd := strings.Replace(sys.Grep(result, chartName), "\t", " ", 2)
 
-	return sys.Awk(cd, 2, " "), nil
+	version := ""
+	slice := strings.Fields(cd)
+	if len(slice) >= 2 {
+		version = slice[1]
+	}
+	return version, nil
```

Comment:
should we actually return an error if the field is not found?

Reply:
Done. Also, removed the replacement of tab character since `strings.Fields` handles tab characters as well.